server/controller/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
```

Comment:
this would save me a line of code. It's not an important comment, but it's more concise.
```suggestion
const router = require('express').router();
```